doc/source/cluster/kubernetes/user-guides/k8s-cluster-setup.md

```diff
@@ -9,9 +9,9 @@ aws-eks-gpu-cluster
 gcp-gke-gpu-cluster
 ```
 
-The KubeRay operator and Ray can run on any cloud or on-prem Kubernetes cluster.
-The simplest way to provision a remote Kubernetes cluster is to use a cloud-based managed service.
-We collect a few helpful links for users who are getting started with a managed Kubernetes service.
+Most KubeRay documentation only requires a local Kubernetes cluster such as [Kind](https://kind.sigs.k8s.io/).
```

Comment:
```suggestion
Most KubeRay documentation only requires a local Kubernetes cluster such as the [kind](https://kind.sigs.k8s.io/) tool.
```

Reply:
Just a suggestion that could keep `kind` in lower case but makes it obvious that it is a the of a tool.

Reply:
Good idea! Perhaps we can address this in a separate PR to update all occurrences of `Kind`.